docs/execution-providers/ROCm-ExecutionProvider.md

```diff
@@ -0,0 +1,71 @@
+---
+title: ROCm (AMD)
+description: Instructions to execute ONNX Runtime with the AMD ROCm execution provider
+parent: Execution Providers
+nav_order: 6
+redirect_from: /docs/reference/execution-providers/ROCm-ExecutionProvider
+---
+
+# ROCm Execution Provider
+{: .no_toc }
+
+The [ROCm](https://docs.amd.com/bundle/ROCm-Installation-Guide-v5.2.3/page/How_to_Install_ROCm.html#_How_to_Install) execution provider uses AMD's Deep Learning graph optimization engine to accelerate ONNX model on AMD GPUs. 
```

Comment:
I think for "graph optimization", only MIGraphX EP will exhibit. More appropriately, this should be:

> The ROCm Execution Provider enables hardware accelerated computation on AMD ROCm-enabled GPUs.

analogs to docs for CUDA EP.